### PR TITLE
use system language for default menu if possible

### PR DESCRIPTION
**Pull request by stonebig**
solves https://github.com/pyzo/pyzo/issues/567

### CODE REVIEW

File: pyzo/util/_locale.py [hunk 1]
```diff
@@ -40,17 +40,36 @@
...
+                        'nl_NL' : 'Dutch',
+                        'pt_BR' : 'Portuguese (BR)',
+                        'pt_PT' : 'Portuguese',
+                        'ru_RU' : 'Russian',
+                        'zh_CN' : 'Simplified Chinese',
+                        'zh_TW' : 'Traditional Chinese'
+                     }
 
 
 def getLocale(languageName):
```
Comment (almarklein) [inline pyzo/util/_locale.py]:
```suggestion
def getLocale(languageName):
```

[the remark this answers is not in the document]
Comment (stonebig) [reply pyzo/util/_locale.py]: I suppose the second version of the change was what you wanted, wasn't it ?

**Comment by stonebig** [comment]
I don't know how suggested changes are resolved in github. let me know if I have to do something else. I see a "resolved conversation button"

File: pyzo/util/_locale.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (almarklein) [reply pyzo/util/_locale.py]: Not sure, I posted that at the time when Github was broken, and I might have sunbmitted something multiple times :)

**Comment by almarklein** [comment]
Thanks!

**Pull request merged**